### PR TITLE
Feat(fx-2960) saved search feature flag

### PR DESCRIPTION
**Pull request by lordkiz**
The type of this PR is: **Feature**

This PR resolves [FX-2960]

### Description
Added the saved search feature flag

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[FX-2960]: https://artsyproduct.atlassian.net/browse/FX-2960

### CODE REVIEW

File: CHANGELOG.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ upcoming:
     - Remove feature flag check for custom size filter - devon
     - Fix iOS app blank screen - ole, mounir
     - Replace "color" filter with "colors" filter - dzmitry tratsiak
+    - Added Saved Feature Flag - kizito
```
Comment (dblandin) [inline CHANGELOG.yml:12]: A minor suggestion for this changelog entry:

- Present tense vs past tense: "Add Saved Search feature flag"

**Review by dblandin — APPROVED** [review]
Looks good! Left a minor suggestion for the changelog entry.

**Review by dimatretyak — APPROVED** [review]
LGTM

File: CHANGELOG.yml (hunk 1 above)
Comment (lordkiz) [reply CHANGELOG.yml]: Thanks. Noted and corrected.

**Pull request merged**